doc/howto/autoinstall-quickstart.rst

```diff
@@ -1,48 +1,39 @@
-.. _autoinstall_quickstart:
+.. _autoinstall_quick-start:
 
 Autoinstall quick start
-***********************
+=======================
 
-The intent of this page is to provide simple instructions to perform an
-autoinstall in a VM on your machine.
+This guide provides instructions to use autoinstall with a current version of Ubuntu for the amd64 architecture in a virtual machine (VM) on your computer.
 
-This page assumes that you are installing a recent Ubuntu release. However,
-for older releases, you can substitute the name of the ISO image but the
-instructions should otherwise be the same.
-
-This page also assumes you are on the AMD64 architecture. There is a
-:ref:`version for s390x<autoinstall-quickstart-s390x>` too.
+For older Ubuntu releases, substitute the version in the name of the ISO image. The instructions should otherwise be the same. See :ref:`<autoinstall-quick-start-s390x>` for instructions on installing on the s390x architecture.
 
 Providing the autoinstall data over the network
-===============================================
+-----------------------------------------------
 
-This method is the one that generalises most easily to doing an entirely
-network-based installation where a machine boots over a network and then is automatically
-installed.
+This method describes a network-based installation. A machine boots over a network and the operating system is automatically installed.
```

Comment:
That's not quite correct.  Netboot is a thing, but what we're describing here isn't netboot.  This text is trying to say that this method of supplying autoinstall is one that we think will also be convenient if someone is doing netboot.

Reply:
How about:

"This method describes a network-based installation. When booting over a network, use this method to deliver autoinstall to perform the installation."

Reply:
Go for it